Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -55,15 +66,13 @@
 
 - type: entity
   id: Nukeops
-  parent: BaseGameRule
+  parent: BaseSubGamemodesRule
```

Comment:
just add the subgamemode stuff into the preset prototype for each of these. the base rule entities don't need these tacked on.

Reply:
like just have a non-abstract SubGamemodesRule that these rules' presets have like with variation?

Reply:
Yeah. Make a new regular gamerule called SubGamemodes or whatever and then add the entity into the rules for the individual presets, like how the scheduler does it.